backend/app/services/flow.py

```
@@ -0,0 +1,1011 @@
+        user_id: int,
+    ) -> FlowInDB:
+        """Create a new Flow configuration."""
+        # Validate flow name uniqueness
+        existing = (
+            db.query(FlowResource)
+            .filter(
+                FlowResource.user_id == user_id,
+                FlowResource.kind == "Flow",
+                FlowResource.name == flow_in.name,
+                FlowResource.namespace == flow_in.namespace,
+                FlowResource.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `FlowResource.is_active:` for truth checks

Replace with `FlowResource.is_active`

(E712)